config/config.go

```diff
@@ -247,6 +247,9 @@ type Adapter struct {
 	// needed for Facebook
 	PlatformID string `mapstructure:"platform_id"`
 	AppSecret  string `mapstructure:"app_secret"`
+
+	// needed for Beachfront adapter with separate endpoint for video
+	VideoEndpoint string `mapstructure:"video_endpoint"`
```

Comment:
We now have the `ExtraAdapterInfo`  config object which is meant to be a JSON string that can contain extra adapter config data. We are encouraging adapters to use this instead of adding on extra parameters to the main object, so we don't balloon out to a dozen new single-adapter fields in this config object.

Reply:
Just a note that beachfront has noted this. Thank you.